docs/getting-started.md

```diff
@@ -0,0 +1,105 @@
+# Getting Started - BAG Bevragen API
+De 'BAG Bevragen' API levert actuele gegevens uit de Basisregistratie Adressen en Gebouwen. Dit betekent dat geen historische gegevens kunnen worden gevraagd en dat gegevens met een eindstatus niet worden geleverd.
+
+De API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
+
+Om aan te sluiten kun je de volgende stappen doorlopen:
+1. [Meld je aan bij het kadaster om toegang te krijgen](#Aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#Functionaliteit)
+3. [Probeer en test de API](#Probeer-en-test-de-API)
+
```

Comment:
Het zou beter zijn als aansluiters eerst de API specificaties bestuderen en een client implementatie bouwen, dan een API key aanvragen voor de acceptatie omgeving en daar de client implementatie eerst testen. Als ze zeker weten dat de client implementatie goed werkt, dan kunnen ze een API key aanvragen voor de productie omgeving. Dit voorkomt onnodige fouten in de productieomgeving.

Reply:
Ik denk niet dat API specificaties bestuderen en client implementatie en dan pas een API key aanvragen voor de acceptatie omgeving een goede volgorde is. Ik kan nl. ook gebruik maken van een bestaande client zoals Postman en de swagger ui. En normaliter ga ik (als dev) de documentatie pas bestuderen als een API niet doet wat ik verwacht.

Reply:
ik maak punt 1 "Meld je aan bij het kadaster om toegang te krijgen tot de testomgeving".
En voeg punt 4 toe "Sluit aan op productie".

Reply:
Dat lijkt me prima.

Reply:
Prima